src/_P004_Dallas.ino

```diff
@@ -9,6 +9,11 @@
 #if defined(ESP32)
   # define ESP32noInterrupts() { portMUX_TYPE mux = portMUX_INITIALIZER_UNLOCKED; portENTER_CRITICAL(&mux)
   # define ESP32interrupts() portEXIT_CRITICAL(&mux); }
+
+// https://github.com/espressif/arduino-esp32/issues/1335
+uint8_t Plugin_004_DS_read_bit(int8_t Plugin_004_DallasPin) ICACHE_RAM_ATTR;
+void Plugin_004_DS_write_bit(uint8_t v, int8_t Plugin_004_DallasPin) ICACHE_RAM_ATTR;
```

Comment:
Make sure that _all_ functions called from these functions also use this compiler flag.
If it is a callback function, then also the variables defined outside these functions should be marked with the `volatile` flag.

Reply:
know about it but in this issue they found some cache bug and marking these functions resolves problem, we should leave it like that